### PR TITLE
Added new pages for Terms of Use and Promo Terms Of Use

### PR DESCRIPTION
**Pull request by eddieferrer**
* CASH-1486 CASH-1240 New pages with content for new TOU and Promo TOU to take effect in Feb. 2020

Will depend on https://github.com/kiva/kiva/pull/7860 for the URL's to be active

### CODE REVIEW

File: src/pages/Legal/PromoTermsOfUse.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+			<tertiary-menu class="show-for-large large-2 tertiary-menu-terms">
+				<ul>
+					<li>
+						<router-link to="/legal/privacy">
+							Privacy Policy
+						</router-link>
+					</li>
+					<li>
+						<router-link to="/legal/terms-of-use-february-2020">
+							Terms of Use
```
Comment (mcstover) [inline src/pages/Legal/PromoTermsOfUse.vue]: Seems like we may need to link to the existing terms from here too...and maybe add the upcoming terms to the original page menu as well. Let's check with Brandon this.

Comment (eddieferrer) [reply src/pages/Legal/PromoTermsOfUse.vue]: Checked with Brandon, his comments on Jira, updated with new commit.

**Comment by mcstover** [comment]
👍

**Comment by BoulderBrains** [comment]
👍

**Comment by BoulderBrains** [comment]
ugg, sorry about that. Accidently closed and commented, instead of just commented

**Comment by BoulderBrains** [comment]
My bad, sorry about that.

**Comment by mcstover** [comment]
👍

**Pull request merged**